doc/nrf/libraries/nrf_security/doc/mbed_tls_header.rst

```diff
@@ -3,23 +3,30 @@
 User-provided Mbed TLS config header
 ####################################
 
-The Nordic Security Module provides a Kconfig interface to control compilation and linking of Mbed TLS and the :ref:`nrf_cc3xx_mbedcrypto_readme` or :ref:`nrf_oberon_readme` libraries.
+The Nordic Security Module provides a Kconfig and CMake based build system to configure the Mbed TLS, :ref:`nrf_cc3xx_mbedcrypto_readme`, and :ref:`nrf_oberon_readme` libraries.
 
-The Kconfig interface and build system ensures that the configuration of nrf_security is valid and working.
-It also ensures that dependencies between different cryptographic APIs are met.
+It is recommended to let this system generate the Mbed TLS configuration header as it enforces dependencies between the libraries.
```

Comment:
```suggestion
The build system can generate the Mbed TLS configuration header, which enforces dependencies between the libraries.
```

Reply:
This would omit the important point that we don't recommend our customers doing this unless they have no other choice.

Reply:
But on separate lines as needed for RST.

Reply:
This would still omit the warning that this procedure (the entirety of this .rst) is not recommended.

Reply:
> Use the build system to generate the Mbed TLS header. This enforces dependencies between the libraries.

Using the imperative here automatically means that is is the recommended way. If there are other ways we _don't_ recommend and aren't entirely necessary to cover some customer cases, maybe we don't need to recommend them at all.

In other words, using a command/imperative automatically carries more weight than something like 'it is recommended'.